python/lsst/meas/base/forcedPhotCcd.py

```diff
@@ -104,12 +106,21 @@ def makeDataRefList(self, namespace):
 
 
 def imageOverlapsTract(tract, imageWcs, imageBox):
-    """Return whether the image (specified by Wcs and bounding box) overlaps the tract
-
-    @param tract: TractInfo specifying a tract
-    @param imageWcs: Wcs for image
-    @param imageBox: Bounding box for image
-    @return bool
+    """Return whether the given bounding box overlaps the tract given a WCS.
+
+    Parameters
+    ----------
+    tract:
+        TractInfo specifying a tract.
+    imageWcs: `lsst.afw.geom.SkyWcs`
+        World coordinate system for the image.
+    imageBox:
+        Bounding box for the image.
```

Comment:
Data types please (and a space before the colon). tract is of type ``` `lsst.skymap.TractInfo` ``` and imageBox is presumably of type ``` `lsst.geom.Box2I` ``` (unless floating boxes are used, in which case 2D)